views/history-modal.html

```diff
@@ -43,6 +43,7 @@
               <td>{{ commit.user.title }}</td>
               <td>
                 <a href="#" data-apos-workflow-review="{{ commit._id }}">Review and Export</a>
+                <a href="#" data-apos-workflow-revert="{{ commit._id }}">Revert</a>
```

Comment:
Please change label to "Revert Draft" and wrap it in `__('')` for localization. (Yes I realize there are other labels unlocalized here, just don't want to add more today.) Our various button macros and such tend to call __() themselves but here we're outputting text directly.

Reply:
done